README.md

```diff
@@ -124,11 +130,12 @@ initialization scripts (and later daemons) to pick from.
 ovs-vsctl set Open_vSwitch . external_ids:k8s-api-server="0.0.0.0:8080"
 ```
 
-Install the following additional packages for python.
+Clone this repository and install the executables.
 
 ```
-pip install netaddr
-pip install eventlet
+git clone https://github.com/openvswitch/ovn-kubernetes
+cd ovn-kubernetes
+pip install -e `pwd`
```

Comment:
agree with Russell, nothing to add.
Not sure if you want to mention that if one intends to do changes to the code the suggested way is
python setup.py develop

I think that if one plans to do changes to a python repo he/she should be already quite familiar with python development